modules/fedimint-wallet/src/lib.rs

```diff
@@ -527,6 +646,18 @@ impl FederationModule for Wallet {
 }
 
 impl Wallet {
+    pub async fn init(
```

Comment:
What do do about `new` vs `init` duplication here? Keep only one?

Reply:
Why call it `init`in the first place?

Reply:
Because `new` was already taken. :D

We have to versions now - one taking the erased, one typed config. It wasn't immediately clear which one (or both) to keep.

Reply:
Deleted `init` now.